fenix/app/src/main/res/values/strings.xml

```diff
@@ -1283,7 +1283,10 @@
 
     <!-- Text shown in the notification that pops up to remind the user that a private browsing session is active for Android 14+ -->
     <string name="notification_erase_title_android_14">Close private tabs?</string>
-    <string name="notification_erase_text_android_14">Tap or swipe this notification to close private tabs.</string>
+    <string name="notification_erase_text_android_14" moz:RemovedIn="124" tools:ignore="UnusedResources">Tap or swipe this notification to close private tabs.</string>
+    <!-- Text shown in the notification description that pops up to remind the user that a private browsing session is active
+    for Android 14+. -->
+    <string name="notification_erase_text_android_14_1">Tap or swipe this notification to close private tabs.</string>
```

Comment:
This looks like the same string!?

Reply:
Oops! I think I was running on auto-pilot and not thinking. Thanks for catching! 😄